### DISCUSSION TITLE
MCA -->  show_rows and  label visualization

### DISCUSSION THREAD

**Discussion by SodaMaremLo** [MaxHalford/prince#168] [Q&A] [answered]
Is there an equivalent of PCA show_rows=False when plotting MCA? 
And how can I visualize the labels in a static way?

 I would like to save an image  without row variables, and with all the column labels visible

**Comment by MaxHalford** [comment, marked as answer]
Hey there @SodaMaremLo. I just implemented this as part of Prince 0.13.0. Here is an example:

```py
import prince

dataset = prince.datasets.load_french_elections()
ca = prince.CA(n_components=3)
ca.fit(dataset)
ca.plot(
    dataset, 
    show_row_markers=True,
    show_row_labels=False,
    show_column_labels=True,
    show_column_markers=False
)
```

![visualization](https://github.com/MaxHalford/prince/assets/8095957/7ee92b07-c11a-4d2b-b8c0-348766957d11)

I hope this works for you!

**Reply by SodaMaremLo** [reply]
Thank you a lot, it worked perfectly!!

**Reply by JmonkG** [reply]
Hello @MaxHalford, great work with your library!. I am having problems executing the before-mentioned code. I am having the following error: 
`[/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827](https://file+.vscode-resource.vscode-cdn.net/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827): UserWarning: Unrecognized encoding channel 'show_row_markers'

[/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827](https://file+.vscode-resource.vscode-cdn.net/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827): UserWarning: Unrecognized encoding channel 'show_row_labels'

[/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827](https://file+.vscode-resource.vscode-cdn.net/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827): UserWarning: Unrecognized encoding channel 'show_column_labels'

[/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827](https://file+.vscode-resource.vscode-cdn.net/Users/anaconda3/lib/python3.11/site-packages/altair/utils/core.py:827): UserWarning: Unrecognized encoding channel 'show_column_markers'
 ` 

And after those warnings I got the following error:`SchemaValidationError: Encoding has no parameter named 'show_row_markers' ` . It happens with every single option that appears on the warnings. I am using VS Code as the IDE on a Mac OS with python version 3.11. Thanks!

**Reply by JmonkG** [reply]
Nevermind, seems there is an issue with Python version `3.11.5`; I switched to `3.11.8` and works perfectly. Perhaps an issue with alt library?
